src/components/radios/conditional-reveal.njk

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout-example.njk
+---
+
+{% from 'radios/macro.njk' import govukRadios %}
+
+{{ govukRadios({
+  "idPrefix": "how-contacted-checked",
+  "name": "how-contacted-checked",
+  "fieldset": {
+    "legend": {
+      "text": "How do you want to be contacted?",
+      "isPageHeading": true
+    }
+  },
+  "items": [
+    {
+      "value": "email",
+      "text": "Email",
+      "conditional": {
+        "html": "<label class=\"govuk-label\" for=\"context-email\">Mobile phone number</label>\n<input class=\"govuk-input govuk-!-width-one-third\" name=\"context-email\" type=\"text\" id=\"context-email\">\n"
```

Comment:
Is it a clearer pattern to use `set` to define this html before the macro? eg

```
{% set emailHtml %}
  <label class="govuk-label" for="context-email">Mobile phone number</label>
  <input class="govuk-input govuk-!-width-one-third" name="context-email" type="text" id="context-email">
{% endset %}
```

Reply:
yes, hence the pr  has

Need to change to set, etc

Reply:
ha sorry! missed that

Reply:
np